infra/resources/prod/function_ioweb_profile.tf

```diff
@@ -0,0 +1,300 @@
+
+  environment = {
+    prefix          = local.prefix
+    env_short       = local.env_short
+    location        = local.location
+    domain          = local.domain
+    app_name        = local.function_ioweb_profile.name
+    instance_number = "01"
+  }
+
+  resource_group_name = azurerm_resource_group.function_web_profile_rg.name
+  health_check_path   = "/api/v1/info"
+  node_version        = 20
+  app_service_plan_id = data.azurerm_app_service_plan.shared_plan_itn.id
+
+  subnet_cidr   = local.cidr_subnet_fn_web_profile
+  subnet_pep_id = data.azurerm_subnet.private_endpoints_subnet.id
+
+  subnet_service_endpoints = {
+    web     = true,
+    storage = true
+  }
+
+  private_dns_zone_resource_group_name = data.azurerm_resource_group.rg_common.name
+
+  virtual_network = {
+    name                = data.azurerm_virtual_network.itn_common.name
+    resource_group_name = data.azurerm_virtual_network.itn_common.resource_group_name
+  }
+
+  app_settings = merge(
+    local.function_ioweb_profile.app_settings
+  )
+  slot_app_settings = merge(
+    local.function_ioweb_profile.app_settings
+  )
+
+  application_insights_connection_string = data.azurerm_application_insights.application_insights.connection_string
+
+  action_group_id = azurerm_monitor_action_group.error_action_group.id
+
+  tags = local.tags
+}
+
+module "function_web_profile_autoscale" {
+  depends_on = [azurerm_resource_group.function_web_profile_rg, module.function_web_profile]
+  source     = "github.com/pagopa/dx//infra/modules/azure_app_service_plan_autoscaler?ref=ab26f57ed34a614fd3fa496c7b521be9ecc88e1b"
+
+  resource_group_name = azurerm_resource_group.function_web_profile_rg.name
+  target_service = {
+    function_app_name = module.function_web_profile.function_app.function_app.name
+  }
+
+  scheduler = {
+    normal_load = {
+      minimum = 3
+      default = 10
+    },
+    maximum = 30
+  }
+
+  scale_metrics = {
+    requests = {
+      statistic_increase        = "Max"
+      time_window_increase      = 1
+      time_aggregation          = "Maximum"
+      upper_threshold           = 2500
+      increase_by               = 2
+      cooldown_increase         = 1
+      statistic_decrease        = "Average"
+      time_window_decrease      = 5
+      time_aggregation_decrease = "Average"
+      lower_threshold           = 200
+      decrease_by               = 1
+      cooldown_decrease         = 1
+    }
+    cpu = {
+      upper_threshold           = 35
+      lower_threshold           = 15
+      increase_by               = 3
+      decrease_by               = 1
+      cooldown_increase         = 1
+      cooldown_decrease         = 20
+      statistic_increase        = "Max"
+      statistic_decrease        = "Average"
+      time_aggregation_increase = "Maximum"
+      time_aggregation_decrease = "Average"
+      time_window_increase      = 1
+      time_window_decrease      = 5
+    }
+    memory = null
+  }
```

Comment:
same as above. if we don't want to change the actual config, this should be aligned. however, I see that the previous config does not have the `divide_by_instance_count` set to `true` on requests but the new module does have it internally.
```suggestion
  scale_metrics = {
    requests = {
      statistic_increase        = "Average"
      time_window_increase      = 1
      time_aggregation          = "Average"
      upper_threshold           = 3000
      increase_by               = 2
      cooldown_increase         = 1
      statistic_decrease        = "Average"
      time_window_decrease      = 5
      time_aggregation_decrease = "Average"
      lower_threshold           = 2000
      decrease_by               = 1
      cooldown_decrease         = 1
    }
    cpu = {
      upper_threshold           = 45
      lower_threshold           = 30
      increase_by               = 2
      decrease_by               = 1
      cooldown_increase         = 1
      cooldown_decrease         = 20
      statistic_increase        = "Average"
      statistic_decrease        = "Average"
      time_aggregation_increase = "Average"
      time_aggregation_decrease = "Average"
      time_window_increase      = 1
      time_window_decrease      = 5
    }
    memory = null
  }
```

Reply:
This will be another autoscale settings, even if applied to the same App Service Plan. The only one that can collide is the metric CPU related, because Request are related to the specific function (public and webprofile) and the CPU is related to the App Service Plan.
I'll align values anyway.